### PR TITLE
Create a README.md

### LINKED ISSUE #2: Create a README.md
The new README should explain the 

- basic objects of the projects, and
- the technologies used,
- how developers can build it

### PR DESCRIPTION
**Pull request by learneradarsh**
I've created README.md with basic structure but don't know what type of data I should explain here.
Please let me know about this.

Closes #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Google Code-in 2017
+
+***
+Google Code-in 2017 Current Leadersis a platform where anyone can see trending organization's projects under going in GCI 2017 with task completed description,communication link,and top contributor's name.
```
Comment (blazeu) [inline README.md]: It is a website to show the current leaders from all organizations in GCI.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Google Code-in 2017
```
Comment (blazeu) [inline README.md]: Google Code-in Leaders

File: README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+***
+
+### Installation
+
+***
+
+### Technologies Used:
+
+***
```
Comment (blazeu) [inline README.md]: Languages, libraries, etc. it's in the repo topic.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+ * [Getting Involved]()
+ * [Support]()
+ * [Authors]()
+ * [License]()
+
+***
+
+### Installation
+
+***
```
Comment (blazeu) [inline README.md]: `npm install` to install dependencies

File: README.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+### Technologies Used:
+
+***
+
+### Usage:
+
+***
```
Comment (blazeu) [inline README.md]: `npm run build` to build the site

File: README.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+### Authors
+
+***
+
+### License
+
+***
```
Comment (blazeu) [inline README.md]: See `package.json`

**Review by blazeu — DISMISSED** [review]

**Comment by blazeu** [comment]
Redo your commit; it should be one commit.
Read these 2 guidelines.

https://coala.io/commit
https://coala.io/git

**Comment by blazeu** [comment]
Ideally a PR shouldn't come from your master branch it can complicate things.

But don't worry, do not close this one. You can discuss about it in our chat.

File: README.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
+# Google Code-in 2017
+
+***
+Google Code-in 2017 Current Leadersis a platform where anyone can see trending organization's projects under going in GCI 2017 with task completed description,communication link,and top contributor's name.
+***
+
+ * [Installation]()
```
Comment (wisn) [inline README.md]: Use `-` instead for a listing. coala will complaints about this.

File: README.md (hunk 1 above)
Comment (blazeu) [reply README.md]: But don't copy and paste my words.

Comment (Monal5031) [inline README.md]: `Leaders is`, also space after commas.

File: README.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+
+### Technologies Used:
+
+***
+
+### Usage:
+
+***
+
+### Getting Involved
```
Comment (Monal5031) [inline README.md]: See other coala repos to get an idea of what type of content you have to add.

File: README.md (hunk 7 above)
Comment (Monal5031) [inline README.md]: Add anchor tags near each subpoint and link them in `()`

**Review by Monal5031 — CHANGES_REQUESTED** [review]
@learneradarsh If you are confused over the content to be added analyse other coala repos and see what and how the content are added there. It might help a lot.

File: README.md (hunk 1 above)
Comment (andrewda) [inline README.md]: We should probably keep lines to <80 chars.

**Review by andrewda — CHANGES_REQUESTED** [review]

File: README.md [hunk 9]
```diff
@@ -0,0 +1,61 @@
+# Google Code-in Leaders
+
+***
+It is a website to show the current leaders from all organizations in GCI.
```
Comment (andrewda) [inline README.md]: Remove "It is"

File: README.md [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+***
+
+ * [Installation](###installation)
```
Comment (wisn) [inline README.md]: No. An anchor start with a single `#`. Read more here <http://assemble.io/docs/Cheatsheet-Markdown.html>

**Review by wisn — CHANGES_REQUESTED** [review]

File: README.md (hunk 9 above)
Comment (andrewda) [reply README.md]: Or replace with "This is" if you prefer

**Comment by learneradarsh** [comment]
done all the requested changes @blazeu 
Please review README.md and let me know

File: README.md [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+You can also drop an email at our [mailing list](https://github.com/coala/coala/wiki/Mailing-Lists).
+***
+### Authors
+You can contact to author for more information.
+[blazeu](mailto:neversleepman@tfwno.gf)
+***
+### License
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+***
+
```
Comment (yukiisbored) [inline README.md]: Remove these useless spaces.

File: README.md [hunk 12]
```diff
@@ -0,0 +1,49 @@
...
+If you would like to be a part of the coala community, you can check out our [Getting In Touch](http://coala.readthedocs.io/en/latest/Help/Getting_In_Touch.html) page or ask us at our active Gitter channel, where we have maintainers from all over the world. We appreciate any help!
+
+We also have a [newcomer guide](http://api.coala.io/en/latest/Developers/Newcomers_Guide.html) to help you get started by fixing an issue yourself! If you get stuck anywhere or need some help, feel free to contact us on Gitter or drop a mail at our [newcomer mailing list](https://groups.google.com/forum/#!forum/coala-newcomers).
+***
+### Support
+Feel free to contact us at our [Gitter channel](https://gitter.im/coala/coala), we'd be happy to help!
+
+If you are interested in commercial support, please contact us on the Gitter channel as well.
+
+You can also drop an email at our [mailing list](https://github.com/coala/coala/wiki/Mailing-Lists).
```
Comment (yukiisbored) [inline README.md]: We don't use mailing lists.

File: README.md [hunk 13]
```diff
@@ -0,0 +1,49 @@
...
+***
+### Getting Involved
+If you would like to be a part of the coala community, you can check out our [Getting In Touch](http://coala.readthedocs.io/en/latest/Help/Getting_In_Touch.html) page or ask us at our active Gitter channel, where we have maintainers from all over the world. We appreciate any help!
+
+We also have a [newcomer guide](http://api.coala.io/en/latest/Developers/Newcomers_Guide.html) to help you get started by fixing an issue yourself! If you get stuck anywhere or need some help, feel free to contact us on Gitter or drop a mail at our [newcomer mailing list](https://groups.google.com/forum/#!forum/coala-newcomers).
+***
+### Support
+Feel free to contact us at our [Gitter channel](https://gitter.im/coala/coala), we'd be happy to help!
+
+If you are interested in commercial support, please contact us on the Gitter channel as well.
```
Comment (yukiisbored) [inline README.md]: No, we don't do commercial support for this project.

File: README.md [hunk 14]
```diff
@@ -0,0 +1,49 @@
+# Google Code-in Leaders
+***
```
Comment (yukiisbored) [inline README.md:2]: Can we remove all of these rulers? They look annoying.

File: README.md [hunk 15]
```diff
@@ -0,0 +1,49 @@
...
+npm install
+```
+***
+### Languages:
+* Node.js
+* JSON
+***
+### Usage:
+```
+npm run build
```
Comment (yukiisbored) [inline README.md:24]: A better documentation on how to open the freshly built site would be nice

File: README.md (hunk 14 above)
Comment (Monal5031) [reply README.md:2]: Yup as we are already using h1 above this, don't need this.

File: README.md [hunk 16]
```diff
@@ -0,0 +1,49 @@
...
+If you are interested in commercial support, please contact us on the Gitter channel as well.
+
+You can also drop an email at our [mailing list](https://github.com/coala/coala/wiki/Mailing-Lists).
+***
+### Authors
+You can contact to author for more information.
+[blazeu](mailto:neversleepman@tfwno.gf)
+***
+### License
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
```
Comment (Monal5031) [inline README.md]: We still haven't added a License to this repo yet.

Comment (Monal5031) [inline README.md:38]: We still haven't added a License to this repo yet.

File: README.md [hunk 17]
```diff
@@ -0,0 +1,49 @@
...
+***
+### Usage:
+```
+npm run build
+```
+***
+### Getting Involved
+If you would like to be a part of the coala community, you can check out our [Getting In Touch](http://coala.readthedocs.io/en/latest/Help/Getting_In_Touch.html) page or ask us at our active Gitter channel, where we have maintainers from all over the world. We appreciate any help!
+
+We also have a [newcomer guide](http://api.coala.io/en/latest/Developers/Newcomers_Guide.html) to help you get started by fixing an issue yourself! If you get stuck anywhere or need some help, feel free to contact us on Gitter or drop a mail at our [newcomer mailing list](https://groups.google.com/forum/#!forum/coala-newcomers).
```
Comment (Monal5031) [inline README.md]: as @yukiisbored mentioned we don't use mailing list, you should remove it from here too.

**Review by Monal5031 — CHANGES_REQUESTED** [review]

**Comment by Monal5031** [comment]
Also @learneradarsh I suggest you rebase and squash your commits too. If you need help with that ping us on zulip/gitter for help. There are 3 PRs for this, close the 2 irrelevant ones.

File: README.md (hunk 16 above)
Comment (andrewda) [reply README.md:38]: And it can be autodetected using shields.io, so may as well do that instead of hardcoding it.

Comment (blazeu) [reply README.md:38]: License is MIT according to package.json

Comment (Monal5031) [reply README.md:38]: Yeah just found a issue for it https://github.com/coala/gci-leaders/issues/6

**Comment by learneradarsh** [comment]
Git rebase and squash done

**Comment by wisn** [comment]
It seems your commits still 13. Rebase and squash should made the commit into 1.

**Comment by Monal5031** [comment]
@learneradarsh Still 2 commits squash them please, btw never press the `update branch` button, it will always create an extra commit. To sync your branch with coala's master do the rebase locally and force push to avoid creation of extra commits.

**Comment by learneradarsh** [comment]
Finally, it's done within one commit.
Please review it and let me know about changes(:

**Review by Monal5031 — CHANGES_REQUESTED** [review]
Please update your branch.
Currently your master is `7 commits behind` coala master which means that you will have to update your master with the latest changes that is the reason your PR shows 30 files changed rather it should only show 1.

**Comment by Mixih** [comment]
You're rebase went awry, and you've somehow integrated the changes of the other ~7 commits that happened after you opened this pr. Don't worry, this is easily fixable. First, do `git reset HEAD^ /path/to/file/` for each file that shouldn't be in you commit. Next do `git commit --amend` to update you commit. After that you can rebase properly from cli using our instructions.

**Comment by jayvdb** [comment]
ping @learneradarsh , GCI is over, but the opportunity to learn continues ;-)
Come onto our Zulip or our Gitter room and someone will step you through fixing this PR.

**Comment by jayvdb** [comment]
I've manually fixed this PR.

**Comment by jayvdb** [comment]
ack https://github.com/coala/gci-leaders/pull/110/commits/429f806a445721a5113fc53cb55e1a1d333c7724

**Review by jayvdb — APPROVED** [review]
Not ideal, but better to have this file than not have this file.

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**